src/vm/perfinfo.h

```diff
@@ -0,0 +1,26 @@
+#ifndef PERFINFO_H
+#define PERFINFO_H
+
+
+#include "sstring.h"
+#include "fstream.h"
+
+class PerfInfo {
+public:
+    PerfInfo(int pid);
+    ~PerfInfo();
+    void LogImage(PEFile* pFile, WCHAR* guid); 
+
+private:
+    CFileStream* m_Stream;
+
+    const char sDelimiter = ';';
```

Comment:
You should put a comment block at the top of this file that describes the file format (and use of delimiter) so that readers understand the file format.